homework/App.java

```diff
@@ -0,0 +1,33 @@
+package homework;
+public class App {
+
+    public static void main(String[] args) {
+    System.out.println("The Area of a 5 by 5 square is ");
+    int a = 5;
+    int b = 9;
+    int c = 2;
+    double d = 3.5;
+    int e = 3;
+    int f = 2;
+    double g = 10.2;
+    int h = 15;
+    double i = 9.0;
+    double j = 15.0;
+    double k = 27.0;
+    double l = 2.0;
+    int m = 10;
+    System.out.println((int)a + b); // Output: 14
+    System.out.println((float)l * d); // Output: 7.0
+    System.out.println((float)i / e); // Output: 3.0
+    System.out.println((float)f + a * g); // Output: 53.0
+    System.out.println((int)h / e * a); // Output: 25
+    System.out.println((float)j / (e * a)); // Output: 1.0
+    System.out.println((float)k / i / e); // Output: 1.0
+    System.out.println((float)k / (i / e)); // Output: 9.0
+    System.out.println((float)k / i / e); // Output: 1.0
+    System.out.print("2");
+    System.out.println((int)+ a); // Output: 25
+    System.out.print('A');
+    System.out.println((int)+ m); // Output: A10
```

Comment:
This, unfortunately, doesn't achieve the equation I presented in the assignment. All this did was print the output side by side, it didn't actually perform the + operation.  You will need to redo this.